portfolio/src/main/webapp/index.html

```diff
@@ -32,15 +32,24 @@ <h1>Heidi Johnson's Portfolio</h1>
           <ul id="comments-container"></ul>
           
           <label for="number-comments">Number of comments:</label>
-            <select onchange="storeNumberComments()" name="number-comments" id="number-comments">
-              <option value="0">0</option>
-              <option value="1">1</option>
-              <option value="2">2</option>
-              <option value="3">3</option>
-              <option value="4">4</option>
-              <option value="5">5</option>
+            <select onclick="getNumCommentsSelected()" name="number-comments" id="number-comments">
```

Comment:
Would it be okay to leave it as onclick? I changed it because when I had it as onchange, it wasn't refreshing when you selected a new value, I think because of the <script> part that stores the value onchange. It still works like before, and fetches the correct number of comments when you select a new number. Thanks for any advice!

Reply:
Nevermind to my above comment, I got it fixed with onchange, thank you!